src/vue/pages/Verification.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="verification">
+    <div
+      class="request-message"
+      :class="`request-message--${kycState}`"
+    >
+      <p class="request-message__content">
+        {{ kycRequestMessage }}
+      </p>
+    </div>
+    <p class="verification__subtitle">
+      {{ 'verification-page.account-type-lbl' | globalize }}
+    </p>
+    <div class="account-type-selector">
+      <router-link
+        :to="vueRoutes.verification.general"
+        class="account-type-selector__item"
+      >
+        <p class="account-type-selector__item-title">
+          {{ 'verification-page.account-type-general-title' | globalize }}
+        </p>
+        <p class="account-type-selector__item-description">
+          {{ 'verification-page.account-type-general-description'
+            | globalize }}
+        </p>
+        <div class="account-type-selector__selected-icon">
+          <i class="mdi mdi-check" />
+        </div>
+      </router-link>
+      <router-link
+        :to="vueRoutes.verification.corporate"
+        class="account-type-selector__item"
+      >
+        <p class="account-type-selector__item-title">
+          {{ 'verification-page.account-type-corporate-title' | globalize }}
+        </p>
+        <p class="account-type-selector__item-description">
+          {{ 'verification-page.account-type-corporate-description'
+            | globalize }}
+        </p>
+        <div class="account-type-selector__selected-icon">
+          <i class="mdi mdi-check" />
+        </div>
+      </router-link>
+    </div>
+    <div class="verification__form">
+      <router-view />
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import { store, vuexTypes } from '@/vuex'
+
+import { vueRoutes } from '@/vue-router/routes'
+
+import { REQUEST_STATES_STR } from '@/js/const/request-states.const'
+import { ACCOUNT_TYPES } from '@tokend/js-sdk'
+
+import { globalize } from '@/vue/filters/globalize'
+
+function verificationGuard (to, from, next) {
```

Comment:
nice. please put a brief description in comment for the future generations why we did stuff like this. every single dev coming here will be asking: "Hey! Why we don’t just use the route guard in routers.js?"

Reply:
Resolved, added